runtime/lua/vim/lsp.lua

```diff
@@ -855,17 +855,20 @@ api.nvim_create_autocmd('VimLeavePre', {
 ---@param params table|nil Parameters to send to the server
 ---@param handler? lsp.Handler See |lsp-handler|
 ---       If nil, follows resolution strategy defined in |lsp-handler-configuration|
----
+---@param unsupported_method_callback? fun(method: string)
```

Comment:
Nit, but given that the method is a parameter to this function, it feels redundant to me to pass it to the callback.

Reply:
Makes sense :+1: